.github/workflows/build-ci.yml

```diff
@@ -58,13 +58,16 @@ jobs:
             -   name: Create MongoDB Replica Set
                 run: |
                     docker run --name mongodb -p 27017:27017 -e MONGO_INITDB_DATABASE=unittest --detach mongo:${{ matrix.mongodb }} mongod --replSet rs --setParameter transactionLifetimeLimitSeconds=5
-                    until docker exec --tty mongodb mongo 127.0.0.1:27017 --eval "db.runCommand({ ping: 1 })"; do
+
+                    if [ "${{ matrix.mongodb }}" = "4.4" ]; then MONGOSH_BIN="mongo"; else MONGOSH_BIN="mongosh"; fi
```

Comment:
Where does `mongosh` come from? IIRC it's not distributed with the server package, so if we download it separately we should be able to use it with 4.4 as well.

Reply:
Oh, I see. Fine with me :)